### PR TITLE
Revise Manual upload of Viral Load results 

### PR DESCRIPTION
**Pull request by dbaluku**
https://app.asana.com/0/1133167201254915/1189864148782897/f

### CODE REVIEW

File: omod/src/main/java/org/openmrs/module/ugandaemrsync/page/controller/ViralLoadUploadPageController.java [hunk 1]
```diff
@@ -13,9 +13,9 @@
 import org.springframework.web.multipart.MultipartFile;
 
 import java.io.BufferedReader;
-import java.io.IOException;
 import java.io.InputStream;
 import java.io.InputStreamReader;
+import java.text.SimpleDateFormat;
 import java.util.*;
```
Comment (slubwama) [inline omod/src/main/java/org/openmrs/module/ugandaemrsync/page/controller/ViralLoadUploadPageController.java:19]: Remove wild card import

File: api/src/main/java/org/openmrs/module/ugandaemrsync/api/impl/UgandaEMRSyncServiceImpl.java [hunk 2]
```diff
@@ -33,11 +33,8 @@
 import java.text.DateFormat;
 import java.text.ParseException;
 import java.text.SimpleDateFormat;
-import java.util.Date;
-import java.util.Collection;
-import java.util.List;
-import java.util.ArrayList;
-import java.util.Locale;
+import java.util.*;
```
Comment (slubwama) [inline api/src/main/java/org/openmrs/module/ugandaemrsync/api/impl/UgandaEMRSyncServiceImpl.java:36]: remove Wild card imports.

Comment (dbaluku) [reply api/src/main/java/org/openmrs/module/ugandaemrsync/api/impl/UgandaEMRSyncServiceImpl.java:36]: Done

Comment (dbaluku) [reply api/src/main/java/org/openmrs/module/ugandaemrsync/api/impl/UgandaEMRSyncServiceImpl.java:36]: Done

**Pull request merged**